dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,343 @@
+<template>
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/1bae632e-233e-4200-94be-56b2da619996)
Ordering on the view page seems a bit random? Is the ordering according to the framework definition?

Reply:
Yes. The ordering of the sections should be identical to that in the excel.

Reply:
Ok, i was just wondering, because "Sonstige" is a typical case which is the last in a list of fields and here it is somewhere in the middle, which i fonud a bit weird. However if this is the ordering of the excel then it is ifne

Reply:
Yes I named that subgroup "Sonstige" because I had no idea how else to call it.  Probably pwc will complain about this, but for now i feel it is ok.   And I did not change any orders in the excel since it is similar to the actualy real questionnaire.

Reply:
Ok, its fine for me to adress this again in a possible update story, after feedback from pwc/users